### DISCUSSION TITLE
CTRL+SHIFT+F5 misbehavior

### DISCUSSION THREAD

**Discussion by luifertorres** [seerge/g-helper#396] [Q&A] [answered]
Since version 0.62.0.0, my external keyboard (Logitech) started to mimic FN+F5 behavior whenever I press CTRL+SHIFT+F5 and this is annoying because all day long I need this binding in my job to restart debugging sessions in coding (I'm a programmer).

Anyone who knows about an existing workaround to get rid of this key binding (CTRL+SHIFT+F5) for performance modes?

![image](https://github.com/seerge/g-helper/assets/9755094/4666a08e-b4ee-4669-acc9-83ddad4eecfd)

**Comment by seerge** [comment]
@luifertorres hello, it's not "since 0.62". It was added by user request ages ago (like in version 0.40 or so) :) to allow control performance modes from external keyboards

**Reply by luifertorres** [reply]
Is there any way to opt-out at this feature?

**Reply by seerge** [reply, marked as answer]
Currently - no, but i can optionally add a setting for that under extra

**Reply by luifertorres** [reply]
It will be great!
I use to program a bit in .NET if you want some collab in this

**Comment by seerge** [comment]
[GHelper.zip](https://github.com/seerge/g-helper/files/11499713/GHelper.zip)
You may check this build, ``config.json`` now supports optional parameter : ``keybind_profile``, set it to 0 to completely disable this binding. Or put a corresponding key-code (for example 117 for F6 to bind it to Ctr+Shift+F6)

```"keybind_profile": 117```
